src/trace-span.js

```diff
@@ -37,7 +39,7 @@ function TraceSpan(name, spanId, parentSpanId) {
  * @param {string} value The new value of the label.
  */
 TraceSpan.prototype.setLabel = function(key, value) {
-  this.labels[key] = '' + value;
+  this.labels[key] = typeof value === 'object' ? util.inspect(value) : '' + value;
```

Comment:
`JSON.stringify(5) === "5"` `JSON.stringify('5') === "\"5\""` It is my understanding that the API will reject these if they do not decode to string values.